app/react/projects/list/components/Locale.js

```diff
@@ -1,7 +1,9 @@
 /* @flow */
 import React from 'react';
 import { Flex, Box } from 'radium-flex';
+import { Link } from 'react-router';
 import locale from 'country-language';
+
 import { calculatePercents } from '../../helpers';
 
 import type { LocaleEntityType } from '../../../types/entityTypes';
```

Comment:
It's not good idea to have Component props type outside Component itself, I think.